logic/src/commonMain/kotlin/com/wire/kalium/logic/sync/SlowSyncWorker.kt

```diff
@@ -10,10 +12,16 @@ class SlowSyncWorker(userSessionScope: UserSessionScope) : UserSessionWorker(use
 
         val result = userSessionScope.users.syncSelfUser()
             .flatMap { userSessionScope.conversations.syncConversations() }
+            .flatMap { userSessionScope.users.syncContacts() }
             .onSuccess { userSessionScope.syncManager.completeSlowSync() }
 
         when ( result ) {
             is Either.Left -> {
+                val failure = result.value
+                println("SYNC FAILED $failure")
```

Comment:
Is this `println(...)` getting printed on the logcat? I'm not 100% positive, but maybe we want to encapsulate it with `Log.e(...)` in case we want to have more control on how to filter these messages against others with more verbosity?